docs/index.html

```diff
@@ -2802,7 +2914,7 @@ <h4>Tutorials</h4>
             <hr class="footer-divider">
             <div class="copyright-cta">
                 <p class="text-uppercase">LazyFox Design - All rights Reserved By <a
-                        href="https://themewagon.com/"><span class="span text-violat ">Themewagon</span></a></p>
+                        href="https://themewagon.com/"><span class="span text-violat ">Themewagon</span></a>, big thanks to: bohning, asgard sings, rakuri, marwin89</p>
```

Comment:
I'd link to our main github profile pages
```
<a href="https://github.com/bohning">bohning</a>
<a href="https://github.com/baklap4">Asgard Sings!</a>
<a href="https://github.com/rakuri255">rakuri255</a>
<a href="https://github.com/marwin89">marwin89</a>
```

Reply:
done: links added